### PR TITLE
COMMERCE-10420 Integration test implementation on the headless-commerce-admin-catalog-test module

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-10420

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/OptionValueResourceTest.java [hunk 1]
```diff
@@ -15,14 +15,269 @@
...
 public class OptionValueResourceTest extends BaseOptionValueResourceTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_user = UserTestUtil.addUser(testCompany);
+
+		_commerceOption = _cpOptionLocalService.addCPOption(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/OptionValueResourceTest.java:53]: this should be _cpOption

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java [hunk 2]
```diff
@@ -15,14 +15,301 @@
...
-@Ignore
 @RunWith(Arquillian.class)
 public class AttachmentResourceTest extends BaseAttachmentResourceTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_commerceCPInstance = CPTestUtil.addCPInstanceWithRandomSku(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java:47]: this should be _cpInstance

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java [hunk 3]
```diff
@@ -15,14 +15,301 @@
...
+			testGroup.getGroupId(), BigDecimal.TEN);
+
+		_commerceDefinition = _commerceCPInstance.getCPDefinition();
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java:50]: _cpDefinition

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java [hunk 4]
```diff
@@ -15,14 +15,301 @@
...
+
+		_commerceProduct = _commerceDefinition.getCProduct();
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java:52]: _cProduct

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/CatalogResourceTest.java [hunk 5]
```diff
@@ -15,14 +15,251 @@
...
-@Ignore
 @RunWith(Arquillian.class)
 public class CatalogResourceTest extends BaseCatalogResourceTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_commerceCPInstance = CPTestUtil.addCPInstanceWithRandomSku(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/CatalogResourceTest.java:49]: same as in the Attachment

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
@smottal please review variable names and change where needed to follow the right pattern

**Comment by smottal** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:stop

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
sent to brian https://github.com/brianchandotcom/liferay-portal/pull/129050

**Pull request closed without merging**